apps/finance/scripts/buidler-hooks.js

```diff
@@ -0,0 +1,20 @@
+let vault
+
+module.exports = {
+  postDao: async function({ _experimentalAppInstaller }, bre) {
+    vault = await _experimentalAppInstaller('vault')
```

Comment:
Just wondering, are we also able to deploy off of a contract from a node module?

It would be nice to use the local Vault dependency here:

1. We want to keep Finance in sync with any local changes here
2. It's nice to avoid going to network if we don't have to

Reply:
You are right. We didn't think about it. We should include it as a new feature. I'll add an issue to track it.

Reply:
Let's also add a TODO note here to remind us to improve it :).